attrs_strict/_error.py

```diff
@@ -1,3 +1,12 @@
+def type_pretty_str(type_):
+    """Returns a string to pretty display a given `type_` to the user."""
+    if (hasattr(type_, "__qualname__") and
```

Comment:
Because, `__qualname__` is a standart attribute which every python class and function has, except primitive type. It consists of the class/function name and the scope in which it is defined. Since `typing.NewType` is just a function, checking its name is the only reliable name to identify it.